play-ws-standalone-json/src/main/scala/play/api/libs/ws/JsonBodyReadables.scala

```diff
@@ -21,7 +21,9 @@ trait JsonBodyReadables {
    * }}}
    */
   implicit val readableAsJson: BodyReadable[JsValue] = BodyReadable { response =>
-    Json.parse(response.bodyAsBytes.toArray)
+    val jsonNode = DefaultObjectMapper.objectMapper.readTree(response.body)
+    DefaultObjectMapper.objectMapper.treeToValue(jsonNode, classOf[JsValue])
+    Json.parse(response.body)
```

Comment:
Lines 24-25 can be removed. We are not using it at all. Line 26 is doing all the trick.

Reply:
This comment is obsolete. I removed the lines 24-25. Line 26 is not 24.